### PR TITLE
Scale infrastructure with user load

### PR DESCRIPTION
**Pull request by franziska-warncke**
scale infrastructure with user load  #71

### CODE REVIEW

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: franziska-nttdata-hesselfeld
+published_date: TBD
+category: cloud
+tags: 
+ - compute
```
Comment (dubrie) [inline docs/catalog/cloud/scale-infrastructure-with-user-load.md:7]: please include `cloud` in the tag list as well.

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ - role:cloud-engineer
+ - size:medium
+---
+
+# Scale infrastructure with user load #
```
Comment (dubrie) [inline docs/catalog/cloud/scale-infrastructure-with-user-load.md:12]: remove the extra `#` at the end

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, scale infrastructure with user laod, will impact two parts:
+
+- `M`: Can reduce the embodied emissions if the application after the optimization uses less hardware overall
+- `E`: Optimizing the application will lead to less electricity consumed over time
```
Comment (dubrie) [inline docs/catalog/cloud/scale-infrastructure-with-user-load.md:30]: nit: We try to keep the E, I, M, and R references in the same order as the equation. Can you flip the E and M entries?

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+## Assumptions
+
+- Optimization needs proficiency in the tools and time to be implemented.
```
Comment (dubrie) [inline docs/catalog/cloud/scale-infrastructure-with-user-load.md:34]: Are there any specific tools that need proficiency for this?

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+## Solution
+
+Scaling back resources during periods of low utilisation eliminates excess capacity and improves efficiency because a unit running at a high utilisation produces energy more efficiently. In addition, reducing the total number of resources used also reduces the amount of embodied carbon.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, scale infrastructure with user laod, will impact two parts:
```
Comment (dubrie) [inline docs/catalog/cloud/scale-infrastructure-with-user-load.md:27]: typo: `load` instead of `laod`

**Review by dubrie — CHANGES_REQUESTED** [review]
Some small changes requested for the initial review

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md (hunk 1 above)
Comment (franziska-nttdata-hesselfeld) [reply docs/catalog/cloud/scale-infrastructure-with-user-load.md:7]: Done

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md (hunk 2 above)
Comment (franziska-nttdata-hesselfeld) [reply docs/catalog/cloud/scale-infrastructure-with-user-load.md]: Done

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md (hunk 3 above)
Comment (franziska-nttdata-hesselfeld) [reply docs/catalog/cloud/scale-infrastructure-with-user-load.md:30]: Done

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md (hunk 5 above)
Comment (franziska-nttdata-hesselfeld) [reply docs/catalog/cloud/scale-infrastructure-with-user-load.md]: Done

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md (hunk 4 above)
Comment (franziska-nttdata-hesselfeld) [reply docs/catalog/cloud/scale-infrastructure-with-user-load.md:35]: Good question! After thinking about it, there is no specific tool which need proficiency for this. So I will delete it. Is it ok, to have a blank assumptions list?

**Comment by franziska-nttdata-hesselfeld** [comment]
I merged #142 into this pattern due to similarity. I think, this PR can be deleted. 
@dubrie Can you please look it up, if this is sufficient, thanks

File: docs/catalog/cloud/scale-infrastructure-with-user-load.md (hunk 4 above)
Comment (dubrie) [reply docs/catalog/cloud/scale-infrastructure-with-user-load.md]: Yes, blank lists are generally OK. You may uncover some during the SME Review section as well.

**Review by dubrie — APPROVED** [review]
Initial Review approval

**Comment by dubrie** [comment]
Moving to SME Review (pending until we get the github ID for the Cloud lead).

**Comment by aoifefitton** [comment]
Initial review completed, ready for SME review @navveenb

**Comment by markus-ntt-seidl** [comment]
LGTM

Maybe it makes sense to link other patterns that are similar? Ex: https://patterns.greensoftware.foundation/catalog/cloud/scale-kubernetes-workloads-based-on-events

**Review by markus-gsf-seidl — APPROVED** [review]

**Pull request merged**